bigqueue_flush_test.go

```diff
@@ -14,97 +14,91 @@ import (
 	"github.com/jonboulle/clockwork"
 )
 
-func TestSetFlushIntervalMutateOpsValidation(t *testing.T) {
-	testDir := path.Join(os.TempDir(), fmt.Sprintf("testdir_%d", rand.Intn(1000)))
+func TestSetPeriodicFlushOpsFail(t *testing.T) {
+	testDir := path.Join(os.TempDir(), fmt.Sprintf("testdir_%v", rand.Intn(1000)))
 	createTestDir(t, testDir)
 	defer deleteTestDir(t, testDir)
 
-	_, err := NewMmapQueue(testDir, SetFlushIntervalMutateOps(0))
+	_, err := NewMmapQueue(testDir, SetPeriodicFlushOps(0))
 	if err != ErrMustBeGreaterThanZero {
 		t.Fatalf("expected error ErrMustBeGreaterThanZero, got: %v", err)
 	}
 }
 
-func TestSetFlushElapsedDurationValidation(t *testing.T) {
-	testDir := path.Join(os.TempDir(), fmt.Sprintf("testdir_%d", rand.Intn(1000)))
+func TestSetPeriodicFlushDurationFail(t *testing.T) {
+	testDir := path.Join(os.TempDir(), fmt.Sprintf("testdir_%v", rand.Intn(1000)))
 	createTestDir(t, testDir)
 	defer deleteTestDir(t, testDir)
 
-	_, err := NewMmapQueue(testDir, SetFlushElapsedDuration(-1))
+	_, err := NewMmapQueue(testDir, SetPeriodicFlushDuration(-1))
```

Comment:
just realized I should've made `SetPeriodicFlushDuration(0)` here. to test against the highest non-allowed value.